src/playbackstrategy/msestrategy.js

```diff
@@ -376,9 +376,11 @@ function MSEStrategy(
   }
 
   function dispatchMaxQualityChangeForKind(kind) {
-    const { qualityIndex, bitrate: bitrateInBps } = mediaPlayer.getTopBitrateInfoFor(kind)
+    const { qualityIndex, bitrate: bitrateInBps } = mediaPlayer.getTopBitrateInfoFor(kind) ?? {}
 
-    DebugTool.dynamicMetric(`${kind}-max-quality`, [qualityIndex, bitrateInBps])
+    if (qualityIndex && bitrateInBps) {
```

Comment:
Is it possible we get a similar issue with `dispatchDownloadQualityChangeForKind`, `emitPlayerInfo` and `onStreamInitialised` above too? they all utilise similar function calls I think.